apps/cookbook/src/app/showcase/badge-showcase/badge-showcase.component.html

```diff
@@ -1,8 +1,67 @@
-<div class="example">
-  <cookbook-badge-example></cookbook-badge-example>
-  <cookbook-code-viewer [html]="exampleHtml"></cookbook-code-viewer>
-  <h4>Properties:</h4>
-  <cookbook-api-description-properties
-    [properties]="properties"
-  ></cookbook-api-description-properties>
-</div>
+<p>
+  Badges are inlines block elements that often are used on top of or close to another component.
+  They can be used with numbers, text and icons. A small badge version also exists which does not
+  support slotted content but can be used to indicate status.
```

Comment:
```suggestion
  support slotted content but can be used to purely indicate status, e.g. a notification dot.
```

Consider something like this, to separate it a bit from the text below for the `md` badge that says _'... can be a good way to indicate status on an element'_